keylime/cmd/create_policy.py

```diff
@@ -589,7 +611,7 @@ def main() -> None:
     policy["meta"]["timestamp"] = str(datetime.datetime.now())
 
     try:
-        ima.validate_runtime_policy(policy)
+        ima.trim_runtime_policy(policy, True)
```

Comment:
IMHO should be a call to the validator, the trim code seems to makes sense only in the converter side

Reply:
Ok, so you are saying if we generate a policy here that is v2 and this policy has no features of a v2 policy then this is fine. Problems with this policy may occur if we were to downgrade keylime to pre-v2  or  want to use the policy on another keylime that only understands v1.

Reply:
Right. My main issue can be summarized as: What to do with policy v3, v4, ..., vN?

The current proposal does not work in the general case, it is an ad-hoc instance for v1 <-> v2 (trim, conflating validation, keeping the version number, etc). Honestly the pre-patched code also does not provide a nice solution to this, but seems to suggest one: work always with the last version.

> to downgrade keylime to pre-v2 or want to use the policy on another keylime that only understands v1.

OK. I accept that using the minimal version can be a valid approach. I will review based on this.